debian/rules

```diff
@@ -23,6 +24,21 @@ else
 endif
 endif
 
+override_dh_gencontrol:
+ifeq ($(DIST_RELEASE),trusty)
+	echo "extra:Depends=libapt-pkg4.12 (>= 1.0.1ubuntu2.23)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),xenial))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.2.30)" >> debian/ubuntu-advantage-tools.substvars
```

Comment:
We could perhaps depend on 1.2.31: https://launchpad.net/ubuntu/+source/apt/1.2.31
There is 1.2.32 in proposed, but doesn't look like it's related to what we are doing here.

Reply:
resolved in commitish e798ea2a0817f421a486b84fba0fe1231de4da8f.   moved to the published version in <series>-updates for each series.